src/main/java/com/jgcolab/mymenu/api/MenuController.java

```diff
@@ -21,22 +22,26 @@
 
 	@Autowired private MenuRepository menuRepository;
```

Comment:
we should not field injection, prefer constructor injection instead

Reply:
Fixed.